worldedit-core/src/main/java/com/fastasyncworldedit/core/queue/implementation/QueueHandler.java

```diff
@@ -50,7 +51,13 @@ public abstract class QueueHandler implements Trimable, Runnable {
             Settings.settings().QUEUE.PARALLEL_THREADS,
             new FaweForkJoinWorkerThreadFactory("FAWE Fork Join Pool Primary - %s"),
             null,
-            false
+            false,
+            Settings.settings().QUEUE.PARALLEL_THREADS,
+            Settings.settings().QUEUE.PARALLEL_THREADS,
+            0,
+            pool -> true,
```

Comment:
Likewise is there a reason for changing the default here? It does "undo" the above change, but is there a reason for going about it this way?

Reply:
This is to prevent RejectedExecutionExceptions when the pool wants to create more threads due to threads waiting for chunks to load

Reply:
Are the threads created for the async chunk load counted for the pool's thread limit then?

Reply:
Yeah it makes sense with the change above